### ISSUE TITLE
Feature request: move mouse to the selected window

### ISSUE THREAD

**Issue by xalt7x** [Leleat/AltTab-Mod#8]
This feature will be helpful for multiple monitors, ultra-wide monitors, and generally when the user needs to input periodically on multiple tiled windows.

We already had a few GNOME extensions that move the cursor to the focused window:
1. [Mouse follows focus](https://github.com/LeonMatthes/mousefollowsfocus). This one with the "[remember last position of cursor inside previous window"](https://github.com/LeonMatthes/mousefollowsfocus/pull/8) PR worked nicely.
The only issue I've had is that when I close the window on one monitor, the cursor sometimes moves to the active window on another monitor. And so movement only after Alt+Tab/Super+Tab Applications/Windows switching seems less intrusive.   
2. [Alt-Tab Move Mouse](https://github.com/buzztaiki/gnome-shell-extension-alt-tab-move-mouse). This one didn't remember the position on a previous window, so I used the first one until GNOME 45 was released.

So, in short, for me, it would be perfect to:
1. Move cursor only after switching using:
    a. Window switcher (Alt+Tab or whatever user/distro sets)
    b. App switcher(Super+Tab)
    c. Group Switcher (Super+Above_Tab, Alt+Above_Tab)
    d. It is not my preference, but probably someone would like it to work only for windows on different displays.
2. Have some position history because, during frequent switching, it could be annoying to always move the cursor from the center/header of the window input position.
